tests/tel2puml/otel_to_pv/data_holders/sql_data_holder/test_sql_dataholder.py

```diff
@@ -703,3 +695,158 @@ def test_find_unique_graphs(
     assert unique_job_ids_per_job_name["test_name_2"] == {
         str(i) for i in range(5)
     }
+
+
+def test_stream_data(
```

Comment:
I think once you test the sub function to this one that was mentioned in a previous comment, some of the tests can probably done for that function.